functions.php

```diff
@@ -41,5 +42,32 @@ public function enqueue_scripts() {
 			wp_enqueue_script( 'wsu-spokane-navigation', get_stylesheet_directory_uri() . '/js/navigation.js', array( 'jquery', 'backbone' ), spine_get_script_version(), true );
 			wp_enqueue_script( 'wsu-spokane-primary', get_stylesheet_directory_uri() . '/js/primary.js', array( 'wsu-spokane-navigation' ), spine_get_script_version(), true );
 	}
+
+	/**
+	 * Modify the color used for the Spine on the front page.
+	 *
+	 * @param string $value  The current Spine color (or other option value).
+	 * @param string $option The current Spine option being filtered.
+	 *
+	 * @return string The modified Spine color.
+	 */
+	public function filter_spine_color( $value, $option ) {
+		if ( 'spine_color' !== $option ) {
+			return $value;
+		}
+
+		$site = get_blog_details();
```

Comment:
Can you do `get_site()` instead? I want to deprecate `get_blog_details()` in WP core one day. :)